Eastern-Research-Group/mywaterway
# Feature/540 data page contact box

## Related Issues:

- [HMW-540](https://jira.epa.gov/browse/HMW-540)

## Main Changes:

- Added a "Contact Us" info box to the bottom of the Data page, similar to the one on the Educators page.

## Steps To Test:

1. Navigate to http://localhost:3000/data.
2. Confirm that the contact box at the bottom of the page is styled correctly.

## TODO:

- [ ] Merge and test #861, and update the production S3 content from there before merging this PR. The changes in this PR can be included in the next release, but the updates in the previous PR can be applied immediately.
